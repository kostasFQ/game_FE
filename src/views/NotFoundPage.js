import React from 'react';
import styles from './assets/NotFoundPage.module.scss';

const NotFoundPage = () => (
  <div className={styles.notFound__container}>
    <div className={styles.notFound__container__status}>404</div>
    <div className={styles.notFound__container__text}>Page not found.</div>
  </div>
)

export default NotFoundPage;