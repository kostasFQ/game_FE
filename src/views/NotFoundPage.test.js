import React from 'react';
import NotFoundPage from './NotFoundPage';
import renderer from 'react-test-renderer';

test('NotFoundPage should be render', () => {
  const button = renderer.create(
    <NotFoundPage />
  );
  let component = button.toJSON();
  expect(component).toMatchSnapshot();
})