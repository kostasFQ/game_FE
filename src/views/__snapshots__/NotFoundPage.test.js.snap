// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should be render 1`] = `
<div
  className="notFound__container"
>
  <div
    className="notFound__container__status"
  >
    404
  </div>
  <div
    className="notFound__container__text"
  >
    Page not found.
  </div>
</div>
`;
